config/dexs.json

```diff
@@ -395,6 +403,10 @@
   "avalancheFujiTestnet": [],
   "bscTestnet": ["0x1b02da8cb0d097eb8d57a175b88c7d8b47997506"],
   "localanvil": [],
+  "linea": [
+    "0x8cFe327CEc66d1C090Dd72bd0FF11d690C33a2Eb",
+    "0x6BDED42c6DA8FBf0d2bA55B2fa120C5e0c8D7891"
```

Comment:
I don't find this contract on Linea. What should it be?
https://explorer.linea.build/address/0x6BDED42c6DA8FBf0d2bA55B2fa120C5e0c8D7891

Reply:
Not sure...I'll remove it for now